src/main.js

```diff
@@ -1,10 +1,42 @@
 import Vue from 'vue'
 import App from './App.vue'
-
 import "bulma/css/bulma.css";
+import { Server } from 'miragejs'
+
+//Some willing text
```

Comment:
Please delete this mysterious comment.

Reply:
Or write a longer comment that explains more, if there's something non-obvious here.